### PR TITLE
mungegithub: AddAssignees also update local obj

### PR DESCRIPTION
**Pull request by apelisse**
Currently, if a munger calls AddAssignee/RemoveAssignees, the next
munger (or even the same munger) is not going to notice the difference
because we don't refresh issue objects. Let's also update the object so
that the mungers can have more up-to-date information

### CODE REVIEW

File: mungegithub/github/github.go [hunk 1]
```diff
@@ -1574,6 +1574,21 @@ func (obj *MungeObject) GetPR() (*github.PullRequest, bool) {
 	return pr, true
 }
 
+// Returns true if the user is in the list of removed assignees
+func userInList(user *github.User, assignees []string) bool {
+	if user == nil {
+		return false
+	}
+
+	login := *user.Login
```
Comment (grodrigues3) [inline mungegithub/github/github.go]: necessary local var?

Comment (apelisse) [inline mungegithub/github/github.go]: Nope, true, I can remove.

File: mungegithub/github/github.go [hunk 2]
```diff
@@ -1574,6 +1574,21 @@ func (obj *MungeObject) GetPR() (*github.PullRequest, bool) {
 	return pr, true
 }
 
+// Returns true if the user is in the list of removed assignees
```
Comment (grodrigues3) [inline mungegithub/github/github.go]: this comment is to specific.  the word `removed` has nothing to do with what this func does

File: mungegithub/github/github.go [hunk 3]
```diff
@@ -1603,6 +1630,11 @@ func (obj *MungeObject) AddAssignee(owner string) error {
 		glog.Errorf("Error assigning issue #%d to %v: %v", prNum, owner, err)
 		return err
 	}
+
+	obj.Issue.Assignees = append(obj.Issue.Assignees, &github.User{
+		Login: &owner,
```
Comment (grodrigues3) [inline mungegithub/github/github.go]: are you sure we don't need any other fields from github.User to be populated

File: mungegithub/github/github.go [hunk 4]
```diff
@@ -1574,6 +1574,20 @@ func (obj *MungeObject) GetPR() (*github.PullRequest, bool) {
 	return pr, true
 }
 
+// Returns true if the user is in the list of removed assignees
+func userInList(user *github.User, assignees []string) bool {
+	if user == nil {
+		return false
+	}
+
```
Comment (grodrigues3) [inline mungegithub/github/github.go]: whitespace

File: mungegithub/github/github.go (hunk 3 above)
Comment (apelisse) [inline mungegithub/github/github.go]: Hopefully not, we don't have anything else anyway? :-/

**Pull request merged**